open-company/open-company-web
# Fix mobile scrolls

Card: https://trello.com/c/GTBI47x9

To test:
- on desktop
- expand a post
- scroll a bit down
- click on the viewers button
- [ ] see WST?
- close the view
- [ ] scroll is preserved?
- now repeat on mobile
- [ ] scroll is preserved?
- now still on mobile add a comment
- click React button in the comment floating menu
- pick a reaction
- [ ] scroll position is reset to the previous position when the picker is dismissed?
- react to a comment on desktop to make sure everything works
